tools/single_use/Normalize_Consent_Data.php

```diff
@@ -122,8 +122,8 @@
         if($status === "no" && !empty($date)) {
             if(empty($withdrawal)) {
                 array_push($errors, "The date of withdrawal is missing for " . $consentName . ":
-                           [ID]     => " . $consentID . "
-                           [CandID] => " . $candID . "
+                           [ID]     => " . $consentID . "		
+                           [CandID] => " . $candID . "		
```

Comment:
I'm a little surprised it passes Travis with the extra whitespace at the end of the line added.